runtime-modules/content-working-group/src/tests.rs

```diff
@@ -958,6 +946,7 @@ struct UpdateCuratorRewardAccountFixture {
 }
 
 impl UpdateCuratorRewardAccountFixture {
+    #[allow(dead_code)] // delete if the method is unnecessary
```

Comment:
what is different about these methods that we are allowing dead_code rather than removing like in other fixtures?